mysql-test/suite/tokudb/t/type_date.test

```diff
@@ -207,19 +209,21 @@ DROP TABLE t1,t2;
 # Bug #31928: Search fails on '1000-00-00' date after sql_mode change
 #
 
+SET sql_mode = 'ALLOW_INVALID_DATES';
```

Comment:
Same here, we are no explicitly allowing invalid dates where before we wanted to see the behavior of using an invalid date.